java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
Looking at the files, I think that was added when I added an error/warning message that used {} for the message construction.